docs/RFCs/package-toml.md

```diff
@@ -0,0 +1,128 @@
+
+The manifest **MAY** contain the following fields:
+
+- `"entry"` - the filename and package-relative path, as a string, to the main entry point of the package. The root for the relative-path is the directory which contains the `Package.toml` file. e.g `"./src/index.js"`
+- `"type"` - if set to `"module"` this will output a [`package.json` with this included](https://nodejs.org/api/esm.html#esm_code_package_json_code_code_type_code_field), and will signal to the node interpreter to run this code an ES6 Module.
+- `"license"` - a valid [SPDX license](https://spdx.org/licenses/) for your code. If a license is not provided then the code is considered to be `UNLICENSED`.
+- `"description"` - a short description of your package
+- `"homepage"` - a URL for your project's homepage
+- `"author"` - who wrote this package
+- `"repository"` - where does this code live
+
+These fields **MAY** or **MAY NOT** be validated by CLI tools or the server-side
+software, depending on the implementation.
+
+Software implementing the entropic federation protocols and `Package.toml` handling **MAY** complain about any of these optional fields as missing (`license` being an obvious candidate).
+
+### Subsections
+
+The manifest file **MAY** contain zero or more subsections as defined below.
+
+#### Dependency Lists
+
+Dependencies lists contain all the resources that are required for
+this package, in various scenarios. The dependencies **MUST** be declared as `"key" = "value"`
+pairs where the `key` is the package's canonical name (as from the top-level section
+of it's `Package.toml` file) and the `value` is a valid SEMVER version or range
+expression that points to a valid package.
+
+Examples:
+
+To specify a specific version:
+
+```
+"toddself@static-pkg.dev/package2toml" = "1.0.4"
+```
+
+To specify all versions within a given major revision:
+
+```
+"toddself@static-pkg.dev/package2toml" = "^1.0.0"
+```
+
+To specify any version
+
+```
+"toddself@static-pkg.dev/package2toml" = "*"
+```
+
+#### Dependency Types
+
+You **MAY** declare dependency types for:
+
+- `[dependencies]` - these are the packages that are required to run this package. They will be installed automatically when this package is required by another package, or when you specifically install the dependencies for this package manually.
+- `[devDependencies]` - these are packages that are required to develop this package. They will be installed only when you install the dependencies for this package manually. If a package depends on the current package, they will **NOT** be installed as part of that dependency graph.
+- `[peerDependencies]` - these packages are rqeuired to run this package, however, they will not be installed and should be additionally depended on or installed by either the parent package or along side this package for development.
```

Comment:
insert generic whining that we need this for compatibility but I hate peerDeps and would love if we just redesigned them.

Reply:
i added peerDeps because it's "supported" but god i hate peer deps so gosh darn much

Reply:
Peer dependencies night also be the only possible type if Entropic will be used to manage packages for ecosystems which do not support multiple versions of the same dependency in their closures (most of them)